README.md

```diff
@@ -184,6 +184,27 @@ defer task.Delete(context)
 err := task.Start(context)
 ```
 
+### Snapshot Plugins
+
+In addition to the built-in Snapshot plugins in containerd, additional external
+plugins can be configured using GRPC. An external plugin is made available using
+the configured name and appears as a plugin alongside the built-in ones.
+
+To add an external snapshot plugin, add the plugin to containerd's config file
+(defaults to `/etc/containerd/config.toml`). The `proxy_plugin.` name will be
```

Comment:
the use of `proxy_plugin.` is a bit confusing here maybe something like:

> The string following `proxy_plugin.` will be used as the name?

Reply:
updated, also the defaults to